tests/unit-core/test-typedarray.c

```diff
@@ -303,6 +303,44 @@ test_typedarray_complex_creation (test_entry_t test_entries[], /**< test cases *
   }
 } /* test_typedarray_complex_creation */
 
+/**
+ * Test get/set/delete property by index.
+ */
+static void test_property_by_index (test_entry_t test_entries[])
+{
+
+  jerry_value_t ja = jerry_create_number (3);
+  jerry_value_t jb = jerry_create_number (6);
+  jerry_value_t jc = jerry_create_number (50);
+  jerry_value_t global_obj_val = jerry_get_global_object ();
+
+  for (uint32_t i = 0; test_entries[i].constructor_name != NULL; i++)
+  {
+    jerry_value_t prop_name = jerry_create_string ((const jerry_char_t *) test_entries[i].constructor_name);
+    jerry_value_t prop_value = jerry_get_property (global_obj_val, prop_name);
```

Comment:
There is no need to access the typedarray via the global object, we have a typedarray constructor function: http://jerryscript.net/api-reference/#jerry_create_typedarray.